app/lib/actions/foreman/host/import_facts.rb

```diff
@@ -8,7 +8,7 @@ def resource_locks
 
         def plan(_host_type, host_name, facts, certname, proxy_id)
           facts['domain'].try(:downcase!)
-          host = ::Host::Base.import_host(host_name, certname, facts, proxy_id)
+          host = ::Host::Base.import_host(host_name, certname, proxy_id)
```

Comment:
Ah, in that case, there needs to be at least `host.save!` at the end, in order to keep that working the same way as before.